src/app/core/common-components/entity-form/entity-form.service.ts

```diff
@@ -267,7 +267,6 @@ export class EntityFormService {
     const form: EntityFormGroup<T> = entityForm.formGroup;
 
     this.checkFormValidity(form);
-
     const updatedEntity = entity.copy() as T;
```

Comment:
```suggestion

    const updatedEntity = entity.copy() as T;
```

undoing change to simplify git changelog